fetch/api/resources/stash-put.py

```diff
@@ -9,6 +11,6 @@ def main(request, response):
     url_dir = u'/'.join(request.url_parts.path.split(u'/')[:-1]) + u'/'
     key = request.GET.first(b"key")
     value = request.GET.first(b"value")
-    request.server.stash.put(key, value, url_dir)
+    request.server.stash.put(key, isomorphic_decode(value), url_dir)
```

Comment:
This coupling is a bit unfortunate and brittle, splitting the getter and setter of the same property across two different files.

Let's add a comment to point out `value` will be json.dump()'ed in stake-take.py so it must be a text string.

Reply:
Comment added. Thank you!